scanpy/preprocessing/_scrublet/__init__.py

```diff
@@ -245,9 +245,11 @@ def _run_scrublet(ad_obs: AnnData, ad_sim: AnnData | None = None):
 
     if batch_key is not None:
         if batch_key not in adata.obs.keys():
-            raise ValueError(
-                "`batch_key` must be a column of .obs in the input annData object."
+            msg = (
```

Comment:
Is there a reason this is failing codecov?  Should we add a test for this?

Reply:
There’s a lot of error messages like this that aren’t covered. I think it’s always good to increase coverage, but touching an error message that wasn’t used shouldn’t be a blocker.

Reply:
Fair enough.